app/views/main.scala.html

```diff
@@ -55,11 +55,11 @@
 @isUserResearchBannerHidden = @{request.cookies.exists((x: Cookie) => x.name == "mdtpurr")}
 
 @authProvider = @{
-    request.session.get(SessionKeys.authProvider)
+    request.session.get(TaiConstants.AuthProvider)
```

Comment:
@vimalsuresh need to check we can log in via verify and gg fine and that the sign out redirects you to ida/signout and gg/signout

Reply:
Tested in local , looks good